packages/core/README.md

```diff
@@ -401,6 +401,37 @@ subsegment.addSubsegment(newSubseg);
 newSubseg.close();
 ```
 
+### Oversampling mitigation 
+To modify the sampling decision at the subsegment level, sampled subsegments can be created using the `addNewSubsegment` and `addSubsegment` APIs, and unsampled subsegments can be created using the `addNewSubsegmentWithoutSampling` and `addSubsegmentWithoutSampling` APIs. 
+
+The code snippet below demonstrates creating a sampled or unsampled subsegment based on the sampling decision of each SQS message processed by Lambda.
+
+```js
+exports.handler = async function(event, context) { 
+    event.Records.forEach(message => {
+
+        const { attributes } = message;
+        let facade = xrayContext.getSegment();
+
+        if(SqsMessageHelper.isSampled(message)){
+          
+          let sampledSubsegment = facade.addNewSubsegment('sqs-subsegment-sampled');
+          console.log("doing batch work - sampled");
```

Comment:
I think calling it batch work made more sense with the fan-in I had before.

Let's just say "processing message" or something?

I do like telling them that it is sampled or not though.